detox/detox.d.ts

```diff
@@ -1287,18 +1287,25 @@ declare global {
         interface NativeElementActions {
             /**
              * Simulate tap on an element
-             * @param point relative coordinates to the matched element (the element size could changes on different devices or even when changing the device font size)
+             * @param point coordinates in the element's coordinate space. Optional (default is the center of the element).
              * @example await element(by.id('tappable')).tap();
              * @example await element(by.id('tappable')).tap({ x:5, y:10 });
              */
             tap(point?: Point2D): Promise<void>;
 
             /**
              * Simulate long press on an element
-             * @param duration (iOS only) custom press duration time, in milliseconds. Optional (default is 1000ms).
+             * @param point coordinates in the element's coordinate space. Optional (default is the center of the element).
+             * @param duration custom press duration time, in milliseconds. Optional (defaults to the standard long-press duration for the platform).
```

Comment:
Please elaborate on the fact that the usage of `duration` should be restricted to cases that are off the standard, namely, when a duration is explicitly specified over the clickability properties of the view (e.g. as with RN gesture handler)

Reply:
✅